### PR TITLE
Log workflow usage

### LINKED ISSUE #1987: Log usage of workflows
We should add logging statements to the workflows in `share/ert/workflows` , they should use the standard logging module, and the level should be `warning` if (we think) they are/should be deprecated, otherwise `info` is fine. The is to facilitate a cleanup of the workflows as many are probably unused.

### PR DESCRIPTION
**Pull request by ManInFez**
Resolves #1987

### CODE REVIEW

**Comment by ManInFez** [comment]
Test this please!

File: ert_shared/cli/workflow.py [hunk 1]
```diff
@@ -4,15 +4,16 @@
...
         msg = "Workflow {} is not in the list of available workflows"
-        logging.error(msg.format(workflow_name))
+        logger.error(msg.format(workflow_name))
         return
     context = workflow_list.getContext()
     workflow.run(ert=ERT.ert, verbose=True, context=context)
     all_successfull = all([v["completed"] for k, v in workflow.getJobsReport().items()])
     if all_successfull:
-        logging.info("Workflow {} ran successfully!".format(workflow_name))
+        logger.info("Workflow {} ran successfully!".format(workflow_name))
```
Comment (oyvindeide) [inline ert_shared/cli/workflow.py:19]: Convert to `f` string while it is being changed?

Comment (sondreso) [reply ert_shared/cli/workflow.py]: Just a note: formatting when logging has performance considerations, and for most cases it's recommended to use for example `%s` in your string and pass the values to log function. See: https://docs.python.org/3/howto/logging.html#optimization

**Review by oyvindeide — APPROVED** [review]
LGTM!

File: ert_shared/cli/workflow.py (hunk 1 above)
Comment (oyvindeide) [reply ert_shared/cli/workflow.py]: It is the recommended practice, but that generally applies to computationally expensive operations. Think formatted string should be ok in this case (without having actually done a benchmark)

**Pull request merged**